### PR TITLE
[Merged by Bors] - Config improvements

### PR DESCRIPTION
**Pull request by qrilka**
Added while working on #119

### CODE REVIEW

File: src/k8-config/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "k8-config"
-version = "1.4.0"
+version = "1.4.1"
```
Comment (sehz) [inline src/k8-config/Cargo.toml:3]: Since this adds new capability and fields, let's bump this to `1.5.0`

File: src/k8-client/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2018"
 name = "k8-client"
-version = "5.3.0"
+version = "5.3.1"
```
Comment (sehz) [inline src/k8-client/Cargo.toml:4]: bump this to `5.4.0` as well

**Review by sehz — CHANGES_REQUESTED** [review]

File: src/k8-client/Cargo.toml (hunk 2 above)
Comment (qrilka) [reply src/k8-client/Cargo.toml]: done

**Review by sehz — APPROVED** [review]
LGTM.  Thanks

**Pull request closed without merging**